packages/shared/src/components/auth/AuthenticationBanner.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ReactElement } from 'react';
+import classed from '../../lib/classed';
+import { OnboardingHeadline } from './OnboardingHeadline';
+import AuthOptions, { AuthDisplay } from './AuthOptions';
+import { AuthTriggers } from '../../lib/auth';
+import { MemberAlready } from '../onboarding/MemberAlready';
+import { useAuthContext } from '../../contexts/AuthContext';
+
+const Container = classed('div', 'flex flex-col w-[23.25rem]');
+
+export function AuthenticationBanner(): ReactElement {
+  const { showLogin } = useAuthContext();
+
+  return (
+    <div
```

Comment:
We might want to use similar setup for other experiments, I already know one coming up soon which had the same idea, so can we have this as a styled component maybe? We can leave the background part out of the styled component though as that might change.

Reply:
Yeah, I agree, though I've actually done that in a separate PR as I felt the need there 😅 

So maybe we can resolve this one out?
https://github.com/dailydotdev/apps/pull/2636/commits/97bde42f56ee86f9062c71670012e17d3f8b2749

Reply:
Ah yes, in the mobile PR, yes that resolves it. I would suggest though a more generic name like BottomBannerContainer. Do we use something similar for cookies banner as well?

Reply:
Cool. Would it be okay for me to tackle that in the other PR?

Reply:
Yup